nwb_explorer/nwb_model_interpreter/nwb_geppetto_mappers.py

```diff
@@ -346,12 +346,14 @@ def build_fields(self, nwbfile):
         ''' Use this function to add the fields you want to see in the Geppetto model '''
         aqc = len(nwbfile.acquisition)
         stim = len(nwbfile.stimulus)
+        experimenter = len(nwbfile.experimenter)
         summary = {}
         if aqc:
             summary['Num. of acquisitions'] = f"{aqc}"
         if stim:
             summary['Num. of stimuli'] = f"{stim}"
-
+        if experimenter:
+            summary['experimenter'] = f"{nwbfile.experimenter[0]}"
```

Comment:
@jrmartin what about `', '.join(nwbfile.experimenter)`?

Reply:
a better general solution would be to transform all string arrays with `', '. join`

Reply:
@filippomc I replace the line with one joining the array contents for the experimenter. I looked at other arrays used by the metadata list, this one seemed to be the only one that needed to be converted into text

Reply:
Thanks @jrmartin